test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -14,6 +14,7 @@
 from voluptuous import ALLOW_EXTRA, PREVENT_EXTRA, All, Any, Invalid, Length, Required, Schema, Self, ValueInvalid
 from ansible.module_utils.six import string_types
 from ansible.module_utils.common.collections import is_iterable
+from ansible.release import __version__
```

Comment:
Since this is a bug fix for ansible-test, can you put it in a separate PR and provide a changelog?

Reply:
reverted the fix